### DISCUSSION TITLE
Customize light background for code, can't get this to work?

### DISCUSSION THREAD

**Discussion by pvcraven** [pradyunsg/furo#413] [Potential Issue] [answered]
I want to change the background color in light mode so it is white and not this:

![image](https://user-images.githubusercontent.com/1463116/162084231-99d49f4d-b148-4224-b0e0-cd082eec39ff.png)

I feel like this should work, but it doesn't seem to. What is the proper way?

```py
html_theme_options = {
    "light_css_variables": {
        "code-color-background": "#FFFFFF",
    },
}
```

**Comment by dgw** [comment, marked as answer]
[`pygments_style` and `pygments_dark_style`](https://pradyunsg.me/furo/customisation/colors/#code-block-styling)

**Comment by pvcraven** [comment]
Oh, I just need to pick a different theme. Got it, thanks.
